integration_tests/e2e/visitTimetable.cy.ts

```diff
@@ -60,22 +60,14 @@ context('View visit schedule timetable', () => {
     visitTimetablePage.scheduleTime(0).contains('10am to 11:30am')
     visitTimetablePage.scheduleType(0).contains('Open')
     visitTimetablePage.scheduleCapacity(0).contains('20 tables')
-    visitTimetablePage.scheduleAttendees(0).within(() => {
-      cy.contains('Enhanced prisoners in:')
-      cy.contains('Group 1')
-      cy.contains('Group 2')
-    })
+    visitTimetablePage.scheduleAttendees(0).contains('Prisoners on Enhanced prisoners in Group 1 and Group 2')
```

Comment:
This sentence doesn't look right, although I appreciate it's 'correct' according to the test data. Could perhaps change the test data on line 32 to something that fits better?